src/handbook/company/security/access-control.md

```diff
@@ -0,0 +1,108 @@
+---
+navTitle: Access Control Policy
+---
+
+# Access Control Policy
+
+| Policy owner   | Effective date |
+| -------------- | -------------- |
+| @knolleary     | 2023-05-01     |
+
+## Purpose
+
+To limit access to information and systems to authorized parties in accordance
+with business objectives.
+
+## Scope 
+
+This policy applies to all FlowForge information systems that process, store, or
+transmit confidential data as defined in the FlowForge [Data Management Policy](./data-management.md).
+
+It applies to all employees of FlowForge and to all external parties with access
+to FlowForge system resources.
+
+## Access Control Policy
```

Comment:
Should also be part of the requirements for vendor selection?

Reply:
Vanta provide a separate `Third-Party Management Policy` that I think address this. Once we get there, we can cross-link.